cmd/kubeadm/app/util/config/testdata/conversion/master/internal.yaml

```diff
@@ -96,6 +96,8 @@ ComponentConfigs:
     ContainerLogMaxFiles: 5
     ContainerLogMaxSize: 10Mi
     ContentType: application/vnd.kubernetes.protobuf
+    DefaultDNSOptions:
```

Comment:
better to have defaults be non-specified.

Reply:
No problem. I will modify this:)